services/dr-manhattan/louvaine.py

```diff
@@ -231,25 +233,17 @@ def get_communities(self):
 
             d1[com]['campaigns'] = l_camps
 
-            l_tags = map(lambda x: x[0], sorted([(k, v) for k, v in d1[com]['hashtags'].iteritems()], key=lambda x: x[1]))
-            if len(l_tags) > 10:
-                d1[com]['hashtags'] = l_tags[:10]
-            else:
-                d1[com]['hashtags'] = l_tags
+            l_tags = map(lambda x: x[0], sorted([(k, v) for k, v in d1[com]['hashtags'].iteritems()], key=iget(1)))
+            d1[com]['hashtags'] = l_tags[:10] # slice
 
-            l_terms = map(lambda x: x[0], sorted([(k, v) for k, v in d1[com]['keywords'].iteritems()], key=lambda x: x[1]))
-            if len(l_terms) > 10:
-                d1[com]['keywords'] = l_terms[:10]
-            else:
-                d1[com]['keywords'] = l_terms
+            # l_terms = map(lambda x: x[0], sorted([(k, v) for k, v in d1[com]['keywords'].iteritems()], key=lambda x: x[1]))
+            l_terms = sorted(list(d1[com]['keywords'].iteritems()), key=iget(1), reverse=1)
```

Comment:
@drJAGartner this wasn't reversed previously to give highest scores first. what am i missing here?